pages/page.py

```diff
@@ -46,7 +45,7 @@ def is_element_present(self, *locator):
             return False
         finally:
             # set back to where you once belonged
-            self.selenium.implicitly_wait(self.testsetup.default_implicit_wait)
+            self.selenium.implicitly_wait(self.timeout)
```

Comment:
I've been hard-coding this to 10 as it should be less than the explicit wait timeout if that timeout is greater than 10. As they're the same, I'm fine with this change. Hopefully you'll remove implicit waits at some point in the future anyway. :smile: